### PR TITLE
Use correct download URLs for Wildfly version >= 25

### PR DESCRIPTION
**Pull request by EmersonPrado**
Fixes [issue #315](https://github.com/voxpupuli/puppet-wildfly/issues/315)

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -283,13 +283,14 @@ Data type:
 ```puppet
 Variant[Pattern[/^file:\/\//],
     Pattern[/^puppet:\/\//],
-    Stdlib::Httpsurl, Stdlib::Httpurl
+    Stdlib::Httpsurl, Stdlib::Httpurl,
+    Undef
   ]
 ```
 
 Source of Wildfly tarball installer.
```
Comment (smortex) [inline REFERENCE.md:291]: Can you adjust this to describe the new behavior?  I think about something like "Custom source of Wildfly tarball installer (automatic when undef)"

**Review by smortex — APPROVED** [review]
That look good!  Maybe just a minor enhancement for the parameter description might make it even better.

File: REFERENCE.md (hunk 1 above)
Comment (EmersonPrado) [reply REFERENCE.md]: Good catch. Changed to "Custom source of Wildfly tarball installer. Defaults to standard Wildfly repositories.", so default behaviour is clear.

**Review by rwaffen — APPROVED** [review]

**Comment by bastelfreak** [comment]
@EmersonPrado it's a bit hard to follow. Does that bump the default version to 25? Will it effect existing installations?

**Comment by EmersonPrado** [comment]
> @EmersonPrado it's a bit hard to follow. Does that bump the default version to 25? Will it effect existing installations?

Description isn't clear indeed.

Wildfly download server changed in version 25. The changes just correct default download link for version >=25, keeping unchanged for previous versions. Doesn't touch anything else.
